### DISCUSSION TITLE
how to process images from google drive api

### DISCUSSION THREAD

**Discussion by michalmachovic** [ImageMagick/ImageMagick#3632] [Help]
hi 
if i want to process image from google drive

why this is working
`wget 'https://drive.google.com/uc?export=download&id=1ydsWevwDxARqrabo5yZEYozez0eZK4K1' -qO- |  convert - -resize 100x100 MGFIN01.png`

but this isnt working ?
`convert "https://drive.google.com/uc?export=download&id=1ydsWevwDxARqrabo5yZEYozez0eZK4K1" -resize 100x100 MGFIN01.png`

that image is publicly available
using second command i am getting
```

convert-im6.q16: not authorized `HTTPS' @ error/delegate.c/InvokeDelegate/1717.
convert-im6.q16: unable to open file `': No such file or directory @ error/constitute.c/ReadImage/544.
convert-im6.q16: no images defined `MGFIN01.png' @ error/convert.c/ConvertImageCommand/3258.
```

thanks

**Comment by snibgo** [comment]
> not authorized `HTTPS'

This means that a setting in your policy.xml does not authorize you to reads images from HTTPS. You can edit that file in a text editor, and remove that line.

**Reply by michalmachovic** [reply]
hi

in policy.xml i have

```
<policy domain="delegate" rights="read" pattern="URL" />
<policy domain="delegate" rights="read" pattern="HTTPS" />
<policy domain="delegate" rights="read" pattern="HTTP" />
```

i guess that should do the trick ?

thanks
